src/main/java/trade/wayruha/whitebit/dto/Trade.java

```diff
@@ -0,0 +1,15 @@
+package trade.wayruha.whitebit.dto;
+
+import lombok.Data;
+
+import java.math.BigDecimal;
+
+@Data
+public class Trade {
+    private BigDecimal tradeID;
```

Comment:
we should not have non-floating point numbers (e.g., int, long) as BigDecimal.  
it's easier to have them as int (or long for any IDs)